libs/contact/src/contact.spec.ts

```diff
@@ -0,0 +1,5 @@
+describe('contact library', () => {
+  it('should be able run the testing suite', () => {
```

Comment:
Wouldn't this technically not fail if the test fails, because it wouldn't run? I think this file should just be removed.